test_webhook.py

```
@@ -0,0 +1,203 @@
+
+
+def test_slack_formatting():
+    """Test Slack message formatting"""
+    print("Testing Slack message formatting...")
+
+    config = WebhookConfig(
+        url="https://hooks.slack.com/services/TEST/WEBHOOK/URL",
+        enabled=False  # Don't actually send during test
+    )
+
+    service = WebhookService(webhook_config=config)
```

Ruff (0.14.10)

Local variable `service` is assigned to but never used

Remove assignment to unused variable `service`

(F841)